platform/darwin/src/MGLStyle.mm

```diff
@@ -214,7 +216,7 @@ - (MGLSource *)memberOfSources:(MGLSource *)object {
 - (MGLSource *)sourceWithIdentifier:(NSString *)identifier
 {
     auto rawSource = self.rawStyle->getSource(identifier.UTF8String);
-    
+
```

Comment:
Let's remove all of these whitespace changes. The convention for Objective-C files (and Xcode's default formatting behavior) is to indent empty lines like the last block.

Reply:
Is there any way to format whitespace of a whole file according Xcode conventions? Adding those back by hand will be super-tedious...

Reply:
Editor ‣ Structure ‣ Re-Indent (⌃I) reindents the selection. You can use it on wide swaths of the code, but probably not the entire file, because the file contains some hand-indented macros.